.ci/run_non_regression_examples.py

```diff
@@ -0,0 +1,55 @@
+import os
+import glob
+from ansys.dpf import core
+import pathlib
+import subprocess
+import sys
+
+os.environ["PYVISTA_OFF_SCREEN"] = "true"
+os.environ["MPLBACKEND"] = "Agg"
+
+actual_path = pathlib.Path(__file__).parent.absolute()
+print(os.path.join(actual_path, os.path.pardir, "examples"))
+
+DPF_SERVER_TYPE = os.environ.get("DPF_SERVER_TYPE", None)
+if DPF_SERVER_TYPE:
+    if DPF_SERVER_TYPE == "INPROCESS":
+        core.SERVER_CONFIGURATION = core.AvailableServerConfigs.InProcessServer
+    elif DPF_SERVER_TYPE == "GRPC":
+        core.SERVER_CONFIGURATION = core.AvailableServerConfigs.GrpcServer
+    elif DPF_SERVER_TYPE == "LEGACYGRPC":
+        core.SERVER_CONFIGURATION = core.AvailableServerConfigs.LegacyGrpcServer
+
+list_tests = [
```

Comment:
@anslpa, @PProfizi I chose these tests as "non-regression" test. Does that seem good to you?

Reply:
@cbellot000 It looks good to me.